projects/1.18/assets/create-clockwork/vs_clockwork/lang/zh_cn.json

```
@@ -0,0 +1,129 @@
+{
+  "itemGroup.clockwork": "VS：发条装置",
+  "itemGroup.vs_clockwork": "VS: 发条装置",
+  "itemGroup.vs_clockwork.clockwork": "VS: 发条装置",
+  "block.vs_clockwork.propeller_bearing": "螺旋桨轴承",
+  "item.vs_clockwork.propeller_bearing": "螺旋桨轴承",
+  "item.vs_clockwork.bluuguu": "神奇粘液",
+  "item.vs_clockwork.bluperglue": "蓝图胶水",
+  "item.vs_clockwork.gravitron": "重力枪",
+  "item.vs_clockwork.creative_gravitron": "创造重力枪",
+  "entity.vs_clockwork.super_glue": "超级胶水",
+  "block.vs_clockwork.physics_infuser": "物理注入器",
+  "block.vs_clockwork.flap_bearing": "襟翼轴承",
+  "item.vs_clockwork.flap_bearing": "襟翼轴承",
+  "block.vs_clockwork.command_seat": "驾驶座",
+  "item.vs_clockwork.command_seat": "驾驶座",
+  "block.vs_clockwork.flap": "襟翼",
+  "item.vs_clockwork.flap": "襟翼",
+  "block.vs_clockwork.wing": "机翼",
+  "item.vs_clockwork.wing": "机翼",
+  "block.vs_clockwork.reactionwheel": "动量轮",
+  "item.vs_clockwork.reactionwheel": "动量轮",
+  "block.vs_clockwork.gyro": "陀螺仪",
+  "item.vs_clockwork.gyro": "陀螺仪",
+  "block.vs_clockwork.redstone_resistor": "红石变速器",
+  "item.vs_clockwork.redstone_resistor": "红石变速器",
+  "block.vs_clockwork.brass_redstone_resistor": "红石变速器（黄铜机壳）",
+  "item.vs_clockwork.brass_redstone_resistor": "红石变速器（黄铜机壳）",
+  "block.vs_clockwork.balloon_casing": "气球外壳",
+  "item.vs_clockwork.balloon_casing": "气球外壳",
+  "block.vs_clockwork.balloon_encased_shaft": "气球包覆轴",
+  "item.vs_clockwork.balloon_encased_shaft": "气球包覆轴",
+  "block.vs_clockwork.combustion_engine": "内燃机",
+  "item.vs_clockwork.combustion_engine": "内燃机",
+  "item.vs_clockwork.pastrymaker": "pastrymaker",
+  "block.vs_clockwork.ballooner": "热气球",
+  "item.vs_clockwork.ballooner": "热气球",
+  "block.vs_clockwork.alt_meter": "高度计",
+  "item.vs_clockwork.alt_meter": "高度计",
+  "block.vs_clockwork.afterblazer": "fterblazer",
+  "item.vs_clockwork.afterblazer": "fterblazer",
+  "block.vs_clockwork.intake": "进气口",
+  "item.vs_clockwork.intake": "进气口",
+  "block.vs_clockwork.ballast": "压舱物",
+  "item.vs_clockwork.ballast": "压舱物",
+  "block.vs_clockwork.goo_block": "黏性方块",
+  "item.vs_clockwork.goo_block": "黏性方块",
+  "block.vs_clockwork.slicker": "滑块",
+  "item.vs_clockwork.slicker": "滑块",
+
+  "block.vs_clockwork.wanderlite_deepslate_ore": "深层漫游矿石",
+  "item.vs_clockwork.wanderlite_deepslate_ore": "深层漫游矿石",
+  "block.vs_clockwork.wanderlite_end_ore": "末地漫游矿石",
+  "item.vs_clockwork.wanderlite_end_ore": "末地漫游矿石",
+  "item.vs_clockwork.wanderlite_crystal": "漫游石水晶",
+  "item.vs_clockwork.wanderlite_cube": "漫游石立方体",
+  "item.vs_clockwork.wanderlite_matrix": "漫游石矩阵",
+  "item.vs_clockwork.incomplete_wanderwand": "未完成的漫游之杖",
+
+
+  "block.vs_clockwork.heat_pipe": "热管",
+  "item.vs_clockwork.heat_pipe": "热管",
+  "item.vs_clockwork.stratodonut": "束流圈",
+  "item.vs_clockwork.physics_infuser": "物理注入器",
+  "item.vs_clockwork.wanderwand": "魔杖",
+  "vs_clockwork.subtitle.physinitialize": "物理注入器：初始化",
+  "vs_clockwork.subtitle.physfinish": "物理已注入",
```

字幕问题请全文通查，并且修改
顺带问一下物理注入是个什么操作流程


```suggestion
  "vs_clockwork.subtitle.physfinish": "物理：注入完毕",
```